distribution/docs/src/main/resources/content/_whitelists/catalog-whitelist.adoc

```diff
@@ -6,39 +6,167 @@
 :order: 01
 :summary: ${ddf-catalog} whitelist.
 
-The following packages have been exported by the ${ddf-catalog} application and are approved for use by third parties:
-
-* `ddf.camel.component.catalog`
-* `ddf.catalog`
-* `ddf.catalog.cache`
-* `ddf.catalog.data`
-* `ddf.catalog.data.metacardtype`
-* `ddf.catalog.event`
-* `ddf.catalog.federation`
-* `ddf.catalog.federation.impl`
-* `ddf.catalog.filter`
-* `ddf.catalog.filter.delegate`
-* `ddf.catalog.operation`
-* `ddf.catalog.plugin`
-* `ddf.catalog.plugin.groomer`
-* `ddf.catalog.pubsub`
```

Comment:
❗️We should more `ddf.catalog.pubsub` and `ddf.catalog.pubsub.tracker` to the _Removed from whitelist_  section.

Reply:
Added to list